performance/SwiftUIPerformanceQuickstart/PerformanceExample/README.md

```diff
@@ -9,8 +9,18 @@ while also allowing you to measure the performance of any piece of code in your
 
 Introduction
 ------------
+The app is centered around image tasks: download, classify, saliency map, and upload. Downloading
+and uploading images is a common task that many developers will want to do, and classifying the
```

Comment:
s/common task that many developers will want to do, and classifying/common developer task. Classifying/

Reply:
Done ✅